index.html

```diff
@@ -52,7 +52,271 @@ <h3>
         <li><a href='https://www.linkedin.com/in/fabriciolatl/'><img alt="linkedin" src='images/linkedin.svg' /></a></li> 
         <li><a href='https://twitter.com/Wafer_lt'><img alt="twitter" src='images/twitter.svg' /></a></li>
       </ul>
+
+      <section class="recent-works-container">
+        <div class="recent-works-title">
+            <h2>My recent Works</h2>
```

Comment:
Your recent-works main section is not matching the Figma template. Kindly fix it.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/178111512-7215cea0-bcca-4c25-b961-353cf69991c0.png) | ![image](https://user-images.githubusercontent.com/33268927/178111474-11b7228c-e54b-4af2-b1a7-167b32ca2b35.png) |